### ISSUE TITLE
Add aframe-event-decorators to utils.

### ISSUE THREAD

**Issue by ngokevin** [aframevr/aframe#3675]
**Description:**

It's useful!

@Adam4lexander

**Comment by Adam4lexander** [comment]
Sure! I'll be happy to get on this.

**Comment by ngokevin** [comment]
Hey, got time to copy it in? Add as a separate module to `utils.js` in like `events.js` or `decorators.js`, then attach it to `AFRAME.utils.bindEvent`.

**Comment by Adam4lexander** [comment]
Sure thing, will look into this within the next few days! Thanks for the clarification of where to put it.

**Comment by dmarcos** [comment]
Any more context about this? It’s hard to follow what this is for. The associated PR is adding a fair amount of code of complexity and the value is not obvious

**Comment by ngokevin** [comment]
Closing in favor of https://github.com/aframevr/aframe/pull/4114 , thanks for the work though!

Benefits are made more obvious in this form.
